### DISCUSSION TITLE
Release v5.3?

### DISCUSSION THREAD

**Discussion by vpuri3** [JuliaGPU/CUDA.jl#2285] [General]

**Comment by maleadt** [comment]
There hasn't been significant work since the previous release. I'd rather wait until a few impending changes (Julia 1.11 support, CUDA 12.4 support) have been merged.

FYI, releasing CUDA.jl involves quite a bit of testing on my side (i.e. on platforms that aren't covered by CI), is why I prefer to wait until there's sufficient reason to release.
